.github/workflows/asr-benchmark.yml

```diff
@@ -37,6 +37,37 @@ jobs:
       - name: Build
         run: swift build -c release
 
+      - name: Download Models if Needed
+        run: |
+          MODELS_DIR="$HOME/Library/Application Support/FluidAudio/Models/parakeet-tdt-0.6b-v3-coreml"
+          VOCAB_FILE="$HOME/Library/Application Support/FluidAudio/parakeet_vocab.json"
+
```

Comment:
I thought the package does this automatically. WE could probably remove now that you've moved it into FLuidInference repo